app/services/config_report_importer.rb

```diff
@@ -0,0 +1,9 @@
+class ConfigReportImporter < ReportImporter
+  def self.authorized_smart_proxy_features
+    ['Puppet', 'Chef']
```

Comment:
I'd suggest moving this to foreman_chef as part of 1.10 compatibility, probably by overriding it?

Would foreman_chef & salt implement their own types of the reports?  It appears not, so I'd guess they'd alias chain it or we'd need to introduce another API.